s3/src/main/scala/akka/stream/alpakka/s3/S3Settings.scala

```diff
@@ -31,27 +31,33 @@ case object DiskBufferType extends BufferType {
 }
 
 object S3Settings {
-  def apply(system: ActorSystem): S3Settings =
-    apply(system.settings.config.getConfig("akka.stream.alpakka.s3"))
+  //def apply(system: ActorSystem): S3Settings = apply(system.settings.config.getConfig("akka.stream.alpakka.s3"))
```

Comment:
I think this signature should be kept, and be available as `apply()(implicit system: ActorSystem)` it's nice to be able to use the same config as your actor system rather than loading a separate config every time you create settings. And then I think the one below should be removed. If someone wants that it can be achieved with the third signature - `S3Settings(ConfigFactory.load())`, but I don't think that is common.